oak-core/src/main/java/org/apache/jackrabbit/oak/query/ast/AstElement.java

```diff
@@ -41,12 +41,12 @@ protected String protect(Object expression) {
         }
     }
 
-    protected String quote(String pathOrName) {
+    protected static String quote(String pathOrName) {
         pathOrName = pathOrName.replaceAll("]", "]]");
         return '[' + pathOrName + ']';
```

Comment:
While we are at it, I would collapse that into a single line, avoiding the reassignment...